flepimop/gempyor_pkg/tests/distributions/test_distributions_common.py

```diff
@@ -1,16 +1,16 @@
 import numpy as np
 import pytest
 from gempyor.distributions import DistributionABC
+from pydantic import PrivateAttr
 
 
 class DummyDistribution(DistributionABC):
     """A simple dummy implementation for testing DistributionABC logic."""
 
     distribution: str = "dummy"
 
-    def __call__(self) -> float | int:
-        """A shortcut for `self.sample(size=1)`."""
-        return self.sample(size=1).item()
+    _lower_bound: float = PrivateAttr(default=0.0)
```

Comment:
where has __call__ gone? should still want the mockobject to have that interface, i'd think?

Reply:
I removed it, the dummy object gets it by subclassing `DistributionABC`. No need to define `__call__` twice.

Reply:
ah, right - forget that got moved into parent class.